azure-pipelines.yml

```diff
@@ -101,11 +105,11 @@ jobs:
     condition: eq(variables['integrationTests'], 'true')
     displayName: Start Docker services
   - task: DotNetCoreCLI@2
-    displayName: dotnet test net8.0
+    displayName: dotnet test
```

Comment:
Previously we separated test runs by target framework so that we could get faster failure notifications. Test runs were a lot longer then, so it _probably_ doesn't matter as much now, but it was kind of nice to have that granularity quickly available. I'm ok with trying it this way again

Reply:
Okay, let's revert when it turns out to be slow in the future. It's 11 minutes now, component builds are a lot faster.